src/api/route-services/pipelines.js

```diff
@@ -0,0 +1,196 @@
+const AWS = require('aws-sdk');
+const crypto = require('crypto');
+const fetch = require('node-fetch');
+const YAML = require('yaml');
+const jq = require('jq-web');
+const config = require('../../config');
+
+
+class PipelinesService {
+  constructor() {
+    this.stepFunctions = new AWS.StepFunctions({
+      region: config.awsRegion,
+    });
+
+    this.clusterInfo = null;
+  }
+
+  async getClusterInformation() {
+    if (!this.clusterInfo) {
+      const eks = new AWS.EKS({
+        region: config.awsRegion,
+      });
+
+      const accountId = await config.awsAccountIdPromise();
+      const info = await eks.describeCluster({ name: `biomage-${config.clusterEnv}` }).promise();
+
+      const { name, endpoint } = info.cluster;
+      const certAuthority = info.cluster.certificateAuthority.data;
+      const roleArn = `arn:aws:iam::${accountId}:role/state-machine-role-${config.clusterEnv}`;
+
+      this.clusterInfo = {
+        accountId, name, endpoint, certAuthority, roleArn,
```

Comment:
you only use `accountId` and `name` from `clusterInfo`, for the rest since they are not used, it is best to delete them

Reply:
`endpoint` and `roleArn` are also used.

Reply:
that's true, I just saw that, all good then